examples/ee-demo/app.js

```diff
@@ -73,9 +73,9 @@ export default class App extends React.Component {
         eeObject: this.state.eeImage,
         visParams: {min: 0, max: 255},
         opacity: 0.5,
-        // Do not display any tile that is not selected
+        // Prevent overlapping tiles from multiple zoom levels
         // https://deck.gl/#/documentation/deckgl-api-reference/layers/tile-layer?section=refinementstrategy-enum-optional-
-        refinementStrategy: 'never'
+        refinementStrategy: 'no-overlap'
```

Comment:
It is worth asking whether this should even be exposed. Not against it, but given the issues with opacity I am not sure the alternative is worth the potential confusion.

Reply:
I would be in favor of a default like `no-overlap`, but include in the documentation that all `TileLayer` props are inherited, so if a user had a reason for forcing an alternative strategy they could override it.

Reply:
I took this out of here and set `no-overlap` in `defaultProps`